R/module_init_data.R

```diff
@@ -72,25 +72,25 @@ srv_init_data <- function(id, data, modules, filter = teal_slices()) {
     data_validated <- if (inherits(data, "teal_data_module")) {
       srv_teal_data(
         "teal_data_module",
-        data = reactive(req(FALSE)), # to .fallback_on_failure to shiny.silent.error
+        data = reactive(teal_data()),
         data_module = data,
         modules = modules,
         validate_shiny_silent_error = FALSE
       )
     } else if (inherits(data, "teal_data")) {
       reactiveVal(data)
     } else if (test_reactive(data)) {
-      .fallback_on_failure(this = data, that = reactive(req(FALSE)), label = "Reactive data")
+      data
     }
 
     if (inherits(data, "teal_data_module")) {
```

Comment:
Please move this to line-73 in order to not repeat this if

Reply:
0d66e860fb6733c1517babfe91dca9778dae19e0